(ToBeClosed)RDKEMW-4206 : subtec-app code rebase

anaswaragk opened the pull request:
Reason for change: Compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:Anaswara KookkalAnaswara_Kookkal@comcast.com

Commit:
RDKEMW-4206 : subtec-app code rebase

Reason for change: Compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:Anaswara KookkalAnaswara_Kookkal@comcast.com

krithikasvraman committed:
changes from RDKAAMP-1100 and RDKAAMP-77

Commit:
RDKEMW-4206 : subtec-app code rebase

Reason for change: Compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:Anaswara KookkalAnaswara_Kookkal@comcast.com

krithikasvraman committed:
RDKEMW-4206: Changes from commits afb011c and 341da1e

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-4206: Changes from commits 81ade1a, 7801b73 and b4fb7c4

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-4206: Changes from commit a95f759 and L2 test updates

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-4206: Fixed unit tests and L2 tests

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

Commit:
updates for gfx package error and ttml vod bg issue

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/subtec-app?pullRequest=32) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/subtec-app?pullRequest=32) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: anaswarakookkal<br/>:white_check_mark: krithikasvraman<br/>:x: krithika.v1<br/><hr/>**krithika.v1** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/subtec-app?pullRequest=32) it.</sub>

krithikasvraman committed:
Revert "updates for gfx package error and ttml vod bg issue"

This reverts commit 359dc53218f3e3c7559bc041a4c374dc0c6580ab.

krithikasvraman committed:
RDKEMW-4206: Updates after testing, review

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 6 files pending identification.
* Protex Server Path: /home/blackduck/github/subtec-app/32/components/generic/subtec-app

* Commit: e4e467846b66e3def6ca3070ae72e3e222b45bb5

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/3959d127968fdaeb20b32f9bbc17f4b5)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: e4e467846b66e3def6ca3070ae72e3e222b45bb5
'

krithikasvraman committed:
Update sample.ttml for test 1077

krithikasvraman committed:
Update sample.ttml for test 1078

krithikasvraman committed:
updated python script for 1078 test

krithikasvraman commented:
code changes are part of different branch, different PR.. closing this
